hpycc/save.py

```diff
@@ -46,13 +62,15 @@ def save_output(connection, script, path_or_buf=None, syntax_check=True,
     return result.to_csv(path_or_buf=path_or_buf, **kwargs)
 
 
-def save_outputs(connection, script, directory=".", filenames=None,
-                 prefix=None, syntax_check=True, delete_workunit=True,
+def save_outputs(connection, script, directory=".", overwrite=True,
```

Comment:
I'm happy to have this debate about whether we should change this, but I don't think this is the forum/ issue for it. Do you want to open up a new issue/ pull request for these changes?

Reply:
I agree we should discuss but I'm going to be an arse and push this out as it makes other elements of our code easier to use (the change occurred once I'd spent an hour cursing over this link not existing while testing a solution!) and, to my mind, is better as it maintains the link between ECL and our output names. We can revert later -if- when I lose.

Reply:
After careful discussion, we've decided to halt this problem by just disabling the blasted thing.